fastlane/Fastfile

```diff
@@ -80,21 +82,39 @@ platform :ios do
 
   desc "Testflight build for Live Channel"
   lane :testflight_live do
+
+    udpated_version_number = increment_version_number(
+      xcodeproj: project,
+      version_number: File.read("VERSION")
+    )
+
+    udpated_build_number = increment_build_number(
```

Comment:
Don't we need to commit and push the incremented version and build number? (the version would still be persisted on git via the file but I guess the build number wouldn't)

Also wonder how we would authenticate git on CI, do you know how android does it?

Reply:
We do need, yeah - it's part of the described flow in the PR message! Not sure why for some reason, it didn't add it.

The authentication is done via a User Key (which isn't ideal if personal 😛) set up on the CircleCI ios-browser project. The git commands will be performed impersonating the user's ssh key owner.